ChildProject/annotations.py

```diff
@@ -268,9 +273,10 @@ def import_annotation(self, annotation, output_filename):
                 raise ValueError("file format '{}' unknown for '{}'".format(annotation_format, raw_filename))
         except Exception as e:
             self.errors.append(str(e))
```

Comment:
not thread-safe. instead, we should have a error column that is filled when importation fails, and we should return the annotation row here instead of appending errors.